vrk-kpa/suomidigi
# SUOPA-800 Twitter

To test: 
- `make fresh drush-cr`
- Open desktop proto: https://xd.adobe.com/view/d3c60a1d-1d76-4a8b-4797-030e9c2e717f-a5f7/
- Log in and edit the front page
- Add new paragraph "Näkymä / Views"
   - Add title **Suomidigin twitter**
   - For view, select Social media
   - For display, select Twitter
- Save the node and compare the new theme on twitter posts and proto (above), make sure they're alike
